### DISCUSSION TITLE
Difficulty training  bottom-up model with 2 mice

### DISCUSSION THREAD

**Discussion by NEMOOutis123** [talmolab/sleap#1387] [Help!]
Hi ,I'm trying to use SLEAP to track mice, but I've run into a problem. I have labeled 2475 of ~130,000 frames in six videoes. I have been using the following skeleton and training parameters, but i still got some mislabeled frames after running inference, how can i improve performance by changing the training parameters:
![@Z(XZ)LMMO9AZ)YPC)8PEHK](https://github.com/talmolab/sleap/assets/139778021/72332d3d-2787-4ab7-89f7-d527c2f536a1)
![P3@A6P06$9%6 J7E~1R1LMJ](https://github.com/talmolab/sleap/assets/139778021/2efc01ce-7b0a-4020-91a7-9a705f666e6f)
![EMZ8)(}6X%)%Q}6~NGQ~$IQ](https://github.com/talmolab/sleap/assets/139778021/c5d9cd07-3f70-4e14-8967-5101228af008)
![~88ZFQOD3746{CV7JUN6%0S](https://github.com/talmolab/sleap/assets/139778021/1ad3adf6-e29d-41ce-89d0-bd74476af363)
![72R%X 6$L0{Y VJN41WTYNR](https://github.com/talmolab/sleap/assets/139778021/948b74d2-021a-4386-9a63-e7082bc1078b)
![SGW8S4UPU7S QMQBNY1GCVA](https://github.com/talmolab/sleap/assets/139778021/45740c12-ef81-4d1b-8023-3c26e36c9c4e)

Our server has four RTX 2080, but GUI can use only one of them. How can i use four of them in SLEAP?
Looking forward to your reply! Thank you!

**Comment by talmo** [comment]
Hi @NEMOOutis123,

Thanks for reaching out! Here's a few suggestions:

- Try decreasing the input scale to 0.5. This seems counterintuitive since you're losing resolution, but the images you have are fairly large and the features fairly coarse, so it might be a better starting point without significant loss of performance.
- Increase the max stride to 64 to enhance the model's ability reason at larger scales
- Decrease the filter rate to 1.5 and increase the filters to 48 to balance the representational capacity in favor of larger scales
- Increase the second loss weight in the bottom-up heads (this is for PAFs) to 10.0 to balance the training towards better PAFs which should help with connecting distal body parts. Decrease it to ~2.0 if you see that the model can't seem to find keypoints at all or only very noisy ones.

I'd try these one by one to see what works best.

Let us know if you have any questions!

Cheers,

Talmo
